src/sage/algebras/quatalg/quaternion_algebra.py

```diff
@@ -102,6 +106,12 @@ class QuaternionAlgebraFactory(UniqueFactory):
       `D` over `K = \QQ`.  Suitable nonzero rational numbers `a`, `b`
       as above are deduced from `D`.
 
+    - ``QuaternionAlgebra(K, L_finite, L_archimedean)``, where `L_finite`
```

Comment:
The names "L_finite" and "L_archimedean" don't give the best idea of what input is expected; names such as e.g. "primes"/"places_fin" (or something similar) and "inv_archimedean" would be a bit clearer.